gcp/api/run_tests.sh

```diff
@@ -14,14 +14,13 @@
 # limitations under the License.
 
 if [ $# -lt 1 ]; then
-  echo "Usage: $0 /path/to/service_account.json"
+  echo "Usage: $0 /path/to/credential.json"
```

Comment:
Is this any of this still necessary at all, given the use of $1 was removed?

Reply:
So this is still useful locally as it allows you to run the tests using ADC. Both the `test_server.py` and `integration_tests.py` expect the credential path as an argument when run locally to map the credentials into the container. We could potentially refactor this to remove the argument but I was avoiding doing it in this PR, but could have my arm twisted otherwise

Reply:
I don't need to be twisting arms :-)